### PR TITLE
[Labware] Add AGenBio_1_troughplate_100000uL_Fl

### PR DESCRIPTION
**Pull request by fderop**
You might want to rename this. There are two types of AGenBio single well reservoirs, one is 100 mL, and the other is 190 mL:
* https://agenbio.en.made-in-china.com/product/pZWaBIPvZMkm/China-Res-190-F-Lad-Consumables-of-Flat-Reservoir.html
* https://en.a-gen.com/RES-100-F-Single-Well-Low-Profile-Design-100ml-Lab-Supplies-Reservoir-Flat-Bottom-Reservoir-pd765693098.html

### CODE REVIEW

File: pylabrobot/resources/agenbio/plates.py [hunk 1]
```diff
@@ -109,3 +109,48 @@ def AGenBio_1_wellplate_Fl(name: str, lid: Optional[Lid] = None) -> Plate:
...
   )
+
+
+def AGenBio_1_WP_Fl_Shallow(name: str, lid: Optional[Lid] = None) -> Plate:
+    """
+    AGenBio Catalog No. RES-100-F
+    - Material: Polypropylene
+    - Max. volume: 100 mL
+    """
+    assert lid is None
```
Comment (rickwierenga) [inline pylabrobot/resources/agenbio/plates.py:120]: why?

Comment (fderop) [reply pylabrobot/resources/agenbio/plates.py]: good question, I deleted the assertion

Comment (rickwierenga) [reply pylabrobot/resources/agenbio/plates.py]: thx

**Comment by rickwierenga** [comment]
thanks!

**Pull request merged**